packages/core/src/dev/index.ts

```diff
@@ -2,7 +2,7 @@ import { IS_PRODUCTION } from '../environment';
 import { AnyInterpreter, DevToolsAdapter } from '../types';
 
 interface DevInterface {
-  services: Set<AnyInterpreter>;
+  behaviors: Set<AnyInterpreter>;
```

Comment:
this property looks suspicious - `AnyInterpreter` is already an actor and not a behavior